src/relay/transforms/fuse_ops.cc

```diff
@@ -234,6 +235,8 @@ class IndexedForwardGraph::Creator : private ExprVisitor {
     Node* node = graph_.node_map.at(call);
     static auto fpattern =
         Op::GetAttr<TOpPattern>("TOpPattern");
+    static auto tshape_dependant = Op::GetAttr<TShapeDataDependant>(
+        "TShapeDataDependant");
```

Comment:
Do we still need this?

Reply:
Yes, this is used to prevent a result of shape function from being fed to a data dependent shape function.